mapfishapp/jsbuild/main.cfg

```diff
@@ -103,3 +103,16 @@ include =
     Styler/lang/de.js
     locale/ext-lang-de.js
 
+[lang/nl.js]
+root =
+    ../src/main/webapp/lib/externals/openlayers/lib
+    ../src/main/webapp/lib/externals/ext/src
+    ../src/main/webapp/lib/externals/geoext/lib
+    ../src/main/webapp/lib/externals/styler/lib
+    ../src/main/webapp/app/js
+#~ first =
+    #~ OpenLayers/Lang/nl.js
+include =
+    GEOR_Lang/nl.js
+    Styler/lang/nl.js
```

Comment:
Did you check these files exists ?

Reply:
Yes.
GEOR_Lang/nl.js, Styler/lang/nl.js and locale/ext-lang-nl.js exist in their respective folders

Reply:
Strange, all subfolders of mapfishapp/src/main/webapp/lib/externals/ are empty in the clone of my fork (openlayers/, ext/, geoext/ and styler/
In my fork, I can't see any files, only what looks like a reference to a georchestra repository (ex: ext @ 74d59e2)

Reply:
You need to `git submodule update --init` in your main repo root.

Reply:
Oh ok, thanks !

Reply:
After git module update --init, I had to add Styler/lang/nl.js.
GEOR_Lang/nl.js and locale/ext-lang-nl.js were already there.

Reply:
Your contribution to Styler/lang/nl.js cannot be part of this PR, since Styler is a git submodule.
You need to fork https://github.com/georchestra/styler and contribute a PR to it, then we'll take care of managing the submodule. Thanks

Reply:
I had not seen https://github.com/georchestra/styler/pull/17 !
Forget my comment then ;-)

Reply:
I had indeed to fork georchestra/styler in order to add the file, I'm learning git...sorry :-)

Normally all required files are exist (or will exist after merging this PR) : 

- **GEOR_Lang.nl** is part of this PR

- **locale/ext-lang-nl.js** is present by default in **../src/main/webapp/lib/externals/ext/src** (nothing to do, translation is already available in the submodule)